test/instream/encoder/line_test.exs

```diff
@@ -207,4 +207,42 @@ defmodule Instream.Encoder.LineTest do
 
     assert expected == Line.encode(points)
   end
+
+  test "with list of strings in tag value" do
+    expected = ~S|disk_free,items=string_a,string_b value=442221834240i|
+
+    points = [
+      %{
+        measurement: "disk_free",
+        tags: %{
+          items: ["string_a", "string_b"]
+        },
+        fields: %{
+          value: 442_221_834_240
+        },
+        timestamp: nil
+      }
+    ]
+
+    assert expected == Line.encode(points)
+  end
+
+  test "with list of atoms in tag value" do
+    expected = ~S|disk_free,items=atom_a,atom_b value=442221834240i|
```

Comment:
This will result in a failed write as a comma is treated as a separator and would require additional quoting:

```
%{error: "unable to parse 'disk_free,items=atom_a,atom_b value=442221834240i': missing tag value"}
```

From that result I think it may not be possible to just complete a list into a value. As tags have to be unique the value would need to be combined and quoted into a single value. As the result of that combination is probably depending on the actual requirements it might be easier to require a user to perform that conversion themselves and not here.

Or can you construct a use case with a conversion not relying on assumptions while not silently dropping any values?